### DISCUSSION TITLE
Using full screen gestures in third party launchers

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [MonwF/customiuizer#194] [Ideas]
As it is seen that we can't use full screen gestures and we need to use button navigations if we use any other third party launcher instead of poco launcher (or mi launcher)
Can you please bypass it

**Comment by MonwF** [comment]
it seems not easy

**Reply by RevealedSoulEven** [reply]
Umm, it's only a major mod left, all other major mods are almost there

**Comment by RevealedSoulEven** [comment]
https://youtu.be/fBn3kNGEE1s


@MonwF have a look. It's something new
